### ISSUE TITLE
Wrong type for Inspect class

### ISSUE THREAD

**Issue by KerberosMorphy** [sbdchd/celery-types#56]
Hi there,

Maybe it's the documentation of Celery that's unclear but for `celery.app.control.Inspect` the method `stats` say [Ref](https://docs.celeryproject.org/en/stable/reference/celery.app.control.html#celery.app.control.Inspect.stats):

> Returns: Dictionary {HOSTNAME: STAT_INFO}.
> Return type: Dict

But inside your code I see:

https://github.com/sbdchd/celery-types/blob/0c29be1a3bbecd8aee6b9619a812eca720bf2988/celery-stubs/app/control.pyi#L33

I have the same issue with `query_task`.

If I follow the rabbit:

`Inspect.stats()`: return `self._request('stats')`
`Inspect._request(...)`: return `self._prepare(...)`
And `Inspect._prepare(...)`:

```python
    def _prepare(self, reply):
        if reply:
            # flatten_reply convert [{"HOSTNAME": Reply}, {"HOSTNAME": Reply}, ...]
            # to {"HOSTNAME": Reply, "HOSTNAME": Reply, ...}
            # so list[dict[str, _Reply]] to dict[str, _Reply]
            by_node = flatten_reply(reply)
            if (self.destination and
                    not isinstance(self.destination, (list, tuple))):
                return by_node.get(self.destination)
            if self.pattern:
                pattern = self.pattern
                matcher = self.matcher
                # Return a dict[str, _Reply]
                return {node: reply for node, reply in by_node.items()
                        if match(node, pattern, matcher)}
            return by_node
```

[Ref for `Inspect._prepare`](https://github.com/celery/celery/blob/56275f5c85247435c14d84807ad254b0f33913c8/celery/app/control.py#L92-L103)

I'm pretty new with django, celery, etc (about a week old) so maybe I just miss something, thanks if someone can explain it to me it'll be more than welcome.

**Comment by sbdchd** [comment]
Thanks for the report, it's probably the type stubs that are incorrect!

**Comment by sbdchd** [comment]
Should be fixed in version 0.11.0
